ingestion_server/ingestion_server/constants/media_types.py

```diff
@@ -4,5 +4,6 @@
 
 AUDIO_TYPE = "audio"
 IMAGE_TYPE = "image"
+3D_MODEL_TYPE = "model_3d"
```

Comment:
Python identifiers cannot start with a number. Please change this.
```suggestion
MODEL_3D_TYPE = "model_3d"
```

Reply:
whoops of course, thank you